src/style.css

```diff
@@ -1,10 +1,93 @@
-* {
-  /* border: 1px solid black; */
-}
-
 body {
   margin: 0;
   padding: 0;
+  background-color: lightblue;
+  font-family: 'Inter', sans-serif;
+  font-family: 'Lato', sans-serif;
+  font-family: 'Poppins', sans-serif;
+  font-family: 'Sanchez', serif;
```

Comment:
- [optional ] Kindly use only one font in the body of the page to make it fit a good design, that's also good practice.